src/olympia/blocklist/tests/test_admin.py

```diff
@@ -1245,14 +1258,18 @@ def test_edit_with_blocklist_create(self):
         log_entry = LogEntry.objects.get()
         assert log_entry.user == user
         assert log_entry.object_id == str(mbs.id)
-        assert log_entry.change_message == json.dumps(
-            [{'changed': {'fields': ['url', 'reason']}}]
-        )
+        change_json = json.loads(log_entry.change_message)
+        # change_message fields are the Field names rather than the fields in django3.2
+        change_json[0]['changed']['fields'] = [
+            field.lower() for field in change_json[0]['changed']['fields']
+        ]
+        assert change_json == [{'changed': {'fields': ['url', 'reason']}}]
 
         response = self.client.get(multi_url, follow=True)
         assert (
-            b'Changed &quot;Pending: guid@, invalid@, second@invalid; '
-            b'new.url; a new reason thats longer than 40 cha...' in response.content
+            f'Changed {FANCY_QUOTE_OR_ENTITY_OPEN}Pending: guid@, invalid@, '
```

Comment:
Because you're using an `in` in this assert, perhaps you could just leave the open quote out of the assertion, as I can see the closed quote is left out? I'm not sure if that compromises the integrity of the test or not, but it would read a bit easier.

Reply:
We kind of want the `'Changed '` at the start too.